docs/installation.md

```diff
@@ -21,6 +21,8 @@ $ cd kraken
 $ pip3 install -r requirements.txt
 ```
 
+**NOTE**: Make sure python3-devel is installed on the system.
```

Comment:
Might be nice to have a link on common ways to download python3-devel to different system types or how to know if it's installed

Reply:
Right, it depends on the linux distribution. Using the containerized version of the Kraken might be the way to go to avoid any issues with the dependencies.